.github/workflows/cypress-test.yml

```diff
@@ -59,23 +82,12 @@ jobs:
           echo 'opensearch_security.readonly_mode.roles: ["kibana_read_only"]' >> ./config/opensearch_dashboards.yml
           echo 'opensearch_security.cookie.secure: false' >> ./config/opensearch_dashboards.yml
           echo 'opensearch_security.multitenancy.enable_aggregation_view: true' >> ./config/opensearch_dashboards.yml
-          yarn start --no-base-path --no-watch &
-          sleep 300
-
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          path: ${{ env.FTR_PATH }}
-          repository: opensearch-project/opensearch-dashboards-functional-test
-          ref: 'main'
-
-      - name: Get Cypress version
-        id: cypress_version
-        run: |
-          echo "::set-output name=cypress_version::$(cat ./${{ env.FTR_PATH }}/package.json | jq '.devDependencies.cypress' | tr -d '"')"
-      
-      - name: Run tests
-        uses: cypress-io/github-action@v2
-        with:
-          working-directory: ${{ env.FTR_PATH }}
-          command: yarn cypress:run-with-security-and-aggregation-view --browser chromium --spec ${{ env.SPEC }}
+          echo 'logging.dest: osd.log' >> ./config/opensearch_dashboards.yml
+          cat ./config/opensearch_dashboards.yml
```

Comment:
Do you want to keep these cats here or were these just for your testing?

Reply:
Let remove these.  If you think they are useful for debugging create a new step that only runs if `ACTIONS_STEP_DEBUG` == true that prints them out

Reply:
Thanks for the review~ I think we can remove that.